### DISCUSSION TITLE
There is an issue with [@types/glidejs__glide] v3.4.1

### DISCUSSION THREAD

**Discussion by elonaire** [DefinitelyTyped/DefinitelyTyped#60033] [Issues with a @types package]
In the index.d.ts line 300, this is definitely a typo - `direction?: 'ltr' | 'rtr' | undefined;` it should be `direction?: 'ltr' | 'rtl' | undefined;`

[a comment by an automated account is not shown]

**Reply by martin-badin** [reply]
Please. Open PR to fix the issue. Thanks

**Comment by elonaire** [comment]
I do not have the permissions. Thank you.

**Comment by MysteryBlokHed** [comment]
I just opened #60036 to fix the type
